docs/release_notes/flare_240.rst

```diff
@@ -0,0 +1,283 @@
+- **cross-site evaluation:** allow clients to evaluate other sites' models.
+
+See `swarm learning <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/swarm_learning>`_ and `client-controlled cyclic <https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/step-by-step/cifar10/cyclic_ccwf>`_ for examples using these client-controlled workflows.
+
+MLFlow and WandB Experiment Tracking Support
+============================================
+We expand our experiment tracking support with MLFLow and Weights & Biases systems.
+The detailed documentation on these features can be found in :ref:`experiment_tracking`, and examples can be found at FL Experiment Tracking with
+`MLFlow <https://nvflare.readthedocs.io/en/main/examples/fl_experiment_tracking_mlflow.html#experiment-tracking-mlflow>`_ and
+`wandb <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/experiment-tracking/wandb>`_.
+
+Multi Configuration File Formats
+================================
+In the 2.4.0 release, we have added support for multiple configuration formats.
+Prior to this release, the sole configuration file format was JSON, which although flexible, was lacking in useful features such as comments, variable substitution, and inheritance.
+
+We added two new configuration formats:
+
+- `Pyhocon <https://github.com/chimpler/pyhocon>`_ - a JSON variant and HOCON (Human-Optimized Config Object Notation) parser for Python, with many desired features
+- `OmegaConf <https://omegaconf.readthedocs.io/en/2.3_branch/>`_ - a YAML based hierarchical configuration
+
+Users have the flexibility to use a single format or combine several formats, as exemplified by config_fed_client.conf and config_fed_server.json.
+If multiple configuration formats coexist, then their usage will be prioritized based on the following search order: .json -> .conf -> .yml -> .yaml
+
+POC Command Upgrade
+===================
+We have expanded the POC command to bring users one step closer to the real deployment process.
+The changes allow users to experiment with deployment options locally, and use the same project.yaml file for both experimentation and in production.
+
+The POC command mode has been changed from "local, non-secure" to "local, secure, production" to better reflect the production environment simulation.
+Lastly, the POC command is now more aligned with common syntax,
+``nvflare poc -<action>`` => ``nvflare poc <action>``
+
+See more details in the :ref:`poc_command` documentation or `tutorial <https://github.com/NVIDIA/NVFlare/tree/main/examples/tutorials/setup_poc.ipynb>`_.
+
+Security Enhancements
+=====================
+
+Unsafe component detection
+--------------------------
+Users now have the capability to define and load an unsafe component checker.
+An event will be fired before building the component, and the checker is used to validate it.
```

Comment:
and the checker will be invoked to validate the component to be built. The checker raises UnsafeJob exception if it fails to validate the component.